playbooks/fundamentals/index.adoc

```diff
@@ -29,6 +29,8 @@ This OpenShift and container fundamentals guide is designed to provide you with
 * link:https://www.openshift.com/promotions/docker-security.html[Docker Security: Using Containers Safely in Production^] to use Docker safely, in both production and development, you need to be aware of the potential security issues and the major tools and techniques for securing container-based systems.
 * link:https://www.openshift.com/promotions/kubernetes.html[Kubernetes: Scheduling the Future at Cloud Scale^] get an introduction to core Kubernetes concepts and components.
 
+* link:https://www.redhat.com/en/blog/red-hat-openshift-container-platform-dns-deep-dive-dns-changes-red-hat-openshift-container-platform-36[DNS Deep Dive^] to learn how OpenShift DNS is working internally.
```

Comment:
I would prefer this link within the OpenShift Guides section instead of the Reading List section.  Would you mind moving this link to the OpenShift guides section below instead?

Will probably need to rebase as well.